docs/typescript.md

```diff
@@ -5,92 +5,44 @@ title: Using TypeScript
 
 import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
 
-[TypeScript][ts] is a language which extends JavaScript by adding type definitions, much like [Flow][flow]. While React Native is built in Flow, it supports both TypeScript _and_ Flow by default.
+[TypeScript][ts] is a language which extends JavaScript by adding type definitions. New React Native projects target TypeScript by default, but also support JavaScript and Flow.
 
-## Getting Started with TypeScript
```

Comment:
This section is overall a lot shorter, since the default new app setup should do this. So it now mostly just links to the environment setup page, with a specific callout for Expo.